src/libapi.mjs.d.ts

```diff
@@ -41,6 +41,9 @@ declare interface SupportedOps {
 }
 
 declare class Context {
+  context(): Promise<any> {
+    throw new Error('Method not implemented.');
```

Comment:
Huh, does this actually compile? I thought `.d.ts` are not allowed to contain any executable code, only types. When would it even be reached?

Reply:
I hadn't noticed that, I genuinely don't know when that slipped in there